hbase-server/src/main/java/org/apache/hadoop/hbase/master/MetaFixer.java

```diff
@@ -86,74 +84,133 @@ void fix() throws IOException {
    * If hole, it papers it over by adding a region in the filesystem and to hbase:meta.
    * Does not assign.
    */
-  void fixHoles(CatalogJanitor.Report report) throws IOException {
-    List<Pair<RegionInfo, RegionInfo>> holes = report.getHoles();
+  void fixHoles(CatalogJanitor.Report report) {
+    final List<Pair<RegionInfo, RegionInfo>> holes = report.getHoles();
     if (holes.isEmpty()) {
-      LOG.debug("No holes.");
+      LOG.info("CatalogJanitor Report contains no holes to fix. Skipping.");
```

Comment:
Did I say it before that CJ will report this every time it runs.... No objection to letting operator know whats up but would rather this part of a single reporting line for CJ instead of a log per task done by the CJ.

Reply:
Good sit, this log line is emitted by `MetaFixer`, speaking of its input. It'll be run just when an operator has invoked hbck2.